src/main/scala/com/databricks/spark/avro/LogicalTypeConverters.scala

```diff
@@ -29,8 +29,18 @@ import org.apache.spark.sql.types._
 
 private object LogicalTypeConverters {
 
-  private[avro] def convertFieldTypeToLogical(): Unit = {
-
+  /**
+  * Converts a supported Spark SQL datatype to the correct avro schema before applying the
+  * logical type schema
+  */
+  private[avro] def convertDataTypeToLogical(dataType: DataType): Schema = {
```

Comment:
This is minor, **convertDataTypeToLogical** sounds bit incomplete ... Logical what? Maybe worth to rename?
Same with **toSql(...)** method name.